### PR TITLE
Delete deprecated `--direction(s)` arguments in the `ask` command

### PR DESCRIPTION
**Pull request by eukaryo**
## Motivation

I want to delete deprecated cli options.

## Description of the changes

I deleted deprecated `--direction` and `--directions` arguments in the `ask` command and related tests.

### CODE REVIEW

**Review by nabenabe0928 — COMMENTED** [review]
I left a minor comment and the PR looks good to me otherwise!

File: tests/test_cli.py [hunk 1]
```diff
@@ -1197,20 +1197,16 @@ def test_storage_upgrade_command_with_invalid_url() -> None:
...
-        (None, None, "TPESampler", '{"multivariate": true}', None),
-        (None, None, None, None, "json"),
-        (None, None, None, None, "yaml"),
+        (None, None, None),
+        ("RandomSampler", None, None),
+        ("TPESampler", '{"multivariate": true}', None),
+        (None, None, "json"),
+        (None, None, "yaml"),
     ],
 )
```
[the remark this answers is not in the document]
Comment (eukaryo) [reply tests/test_cli.py]: I have modified the code in commit https://github.com/optuna/optuna/pull/5405/commits/9e8b4fc6c6e9b9bd856f71eea4444e0102b91b23 .

**Review by not522 — APPROVED** [review]
LGTM!

**Review by nabenabe0928 — APPROVED** [review]
LGTM!

**Pull request merged**